contributions.md

```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
 4. Additional content describing common pitfalls, challenges, or limitations teams commonly encounter when applying certain practices or capabilities.
 5. Typos or grammatical fixes.
 
-If you chose to contribute a new practice try to follow our pre-established structure:
+If you chose to contribute a new practice, please try to follow our pre-established structure:
 
 ```markdown
-# Title Of Practice
+# Title of Practice
 
-A 2-3 sentence description of the practice followed by a sentence or two describing the main benefits of adopting this practice. Try to keep it brief.
+A 2-3 sentence description of the practice, followed by a sentence or two describing the main benefits of adopting this practice. Try to keep the text brief, relatable, and motivational.
 
 ## Nuance
 
-This section outlines 2-4 common pitfalls, challenges, or limitations teams commonly encounter when adopting this practice. Each nuance should have a subheader and description.
+This next section outlines 2-4 common pitfalls, challenges, or limitations that teams commonly encounter when adopting this practice. Each nuance should have a title (styled as a subhead) and a brief (3-5 sentences) description.
```

Comment:
I did a quick check of practices edited thus far: 
- Funtional Core, Imperative Shell has 6 nuances
- Reduce Coupling has 6 nuances
- Run Pair Programming has 4 nuances

Edit first line here to: “This next section outlines ideally 3-4 common pitfalls…” ?